network-dns-search.ks.in

```diff
@@ -0,0 +1,36 @@
+#version=DEVEL
+#test name: network-dns-search
+# Test domain name search properties set from kickstart
+
+# Use defaults.
+%ksappend repos/default.ks
+%ksappend common/common_no_payload.ks
```

Comment:
This includes
```
network --bootproto=dhcp
```
which is not wanted for a networking test setting the network command, instead, I'd use this ksappend battery:
```
%ksappend common/common.ks
%ksappend repos/default.ks
%ksappend l10n/default.ks
%ksappend users/default.ks
%ksappend storage/default.ks
%ksappend payload/default_packages.ks
```